### PR TITLE
[common] Increase tensor_size_extra_limit to 240

### PR DESCRIPTION
**Pull request by anyj0527**
- Making the total size limit to be 256 (16 + 240)
TODO: Set NNS_TENSOR_SIZE as 256 and update other source codes accordingly

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: gst/nnstreamer/include/tensor_typedef.h [hunk 1]
```diff
@@ -35,7 +35,7 @@
 #define NNS_TENSOR_RANK_LIMIT_PREV (4)
 #define NNS_TENSOR_SIZE_LIMIT	(16)
 #define NNS_TENSOR_SIZE_LIMIT_STR	"16"
-#define NNS_TENSOR_SIZE_EXTRA_LIMIT (100)
+#define NNS_TENSOR_SIZE_EXTRA_LIMIT (240)
```
Comment (jaeyun-jung) [inline gst/nnstreamer/include/tensor_typedef.h:38]: Suggestion: how about updating NNS_TENSOR_SIZE_LIMIT (256)?

e.g.,
NNS_TENSOR_SIZE_LIMIT_STATIC (16)
NNS_TENSOR_SIZE_EXTRA_LIMIT (NNS_TENSOR_SIZE_LIMIT - NNS_TENSOR_SIZE_LIMIT_STATIC)

Comment (anyj0527) [reply gst/nnstreamer/include/tensor_typedef.h:45]: You're right. That is obviously proper way of defining the value. I added a todo comment.
I'll do the work after the ACR of ml-api is done.

**Review by wooksong — APPROVED** [review]
LGTM!

**Review by songgot — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Pull request merged**